src/app/(auth)/signin/page.tsx

```diff
@@ -0,0 +1,117 @@
+"use client";
+import { useFormik } from "formik";
+import { string } from "yup";
+import AuthForm from "@/components/organism/auth-form";
+import { login } from "@/models/services/kanvas/mutations/auth";
+import { useRouter } from "next/navigation";
+
+const emailSchema = string().required().email();
+const passwordSchema = string().required();
+
+function useSignInPage() {
+  const router = useRouter();
+  const {
+    handleBlur,
+    handleChange,
+    handleSubmit,
+    errors,
+    values,
+    touched,
+    isSubmitting,
+    isValid,
+  } = useFormik({
+    initialValues: { email: "", password: "" },
+    onSubmit: async ({ email, password }, { setSubmitting }) => {
+      try {
+        await login(email, password);
+        router.push("/search");
+      } catch (e: any) {
+        alert(e.message);
+      } finally {
+        setSubmitting(false);
+      }
+    },
+    validate: ({ email, password }) => {
+      try {
+        emailSchema.validateSync(email);
+      } catch (e: any) {
+        return { email: e.message };
+      }
+      try {
+        passwordSchema.validateSync(password);
+      } catch (e: any) {
+        return { password: e.message };
+      }
+    },
+  });
+  return {
+    models: {
+      isValid,
+      isSubmitting,
+      values,
+      errors,
+      touched,
+    },
+    operations: {
+      handleBlur,
+      handleChange,
+      handleSubmit,
+    },
+  };
+}
+
+export default function SignInPage() {
+  const {
+    models: { isValid, isSubmitting, values, errors, touched },
+    operations: { handleBlur, handleChange, handleSubmit },
+  } = useSignInPage();
+  return (
+    <main className="flex flex-col gap-[50px] items-center">
+      <AuthForm
+        header="Sign in to your account"
```

Comment:
Change: Remember we are using internationalization, we we have to move all the text to the translate folder an call it with the translate function